traffic_ops/app/conf/cdn.conf

```diff
@@ -47,6 +47,7 @@
         "pass_reset_path" : "user",
         "user_register_path" : "user"
     },
+    "portal_url" : "https://traffic-control-cdn.readthedocs.io/en/latest/overview/traffic_portal.html",
```

Comment:
there's already a "portal" key in this file. you want to just put this under that? i.e. portal.docs?